ext/standard/tests/file/windows_mb_path/test_cp932_2.phpt

```diff
@@ -48,8 +48,8 @@ int(25)
 bool(true)
 string(25) "writing to an mb filename"
 Active code page: 932
-getting basename of %s\�e�X�g�}���`�o�C�g�E�p�X33
-string(%d) "�e�X�g�}���`�o�C�g�E�p�X33"
+getting basename of %s\�e�X�g�}���`�o�C�g�E�p�X33
+string(%d) "�e�X�g�}���`�o�C�g�E�p�X33"
 bool(true)
-string(%d) "%s\�e�X�g�}���`�o�C�g�E�p�X33"
+string(%d) "%s\�e�X�g�}���`�o�C�g�E�p�X33"
```

Comment:
You broke the encoding of this (and other) tests.

Reply:
I know I know :wink: